gdx/src/com/badlogic/gdx/Version.java

```diff
@@ -46,20 +46,36 @@ public class Version {
 		}
 	}
 
+	/** @param major
```

Comment:
If their is no parameter description things like `@param major` are in my opinion pointless.
Just noting the return is imo fine here